.ci/packer_cache.sh

```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/env bash
+
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+
+# This script takes responsibility to create a minikube cluster. It has all
+# of the necessary default settings so that no environment variable has to
+# be specified.
+
```

Comment:
We recommend adding `set -eou pipefail` in the beginning of the script to fail if we encounter an error during packer cache

Reply:
Fixed